src/library/serato/seratofeature.cpp

```diff
@@ -450,10 +450,25 @@ QString parseDatabase(mixxx::DbConnectionPoolPtr dbConnectionPool, TreeItem* dat
     databaseRootDir.cdUp();
```

Comment:
This line is unnecessary now, right?

Reply:
No. `QDir:'root()` is a static method and always returns the system drive letter. I think you can move your home directory onto a different drive on Windows, right?